transform/macros/tooling/grant_database_creation_to_dbt_prd.sql

```diff
@@ -0,0 +1,14 @@
+{# This macro grants 'create database' permission to transformer_dbt_prd. #}
```

Comment:
We can get rid of this once we change who creates the db which should be SYSADMIN